client/core/core.go

```diff
@@ -2875,6 +2877,46 @@ func (c *Core) tipChange(assetID uint32, nodeErr error) {
 	c.updateBalances(counts)
 }
 
+func (c *Core) ShutdownWithActiveOrders() bool {
+	c.connMtx.Lock()
+	defer c.connMtx.Unlock()
+	ok := true
+	for _, dc := range c.conns {
+		if dc.hasActiveOrders() {
+			ok = false
+			break
+		}
+	}
+	if !ok {
+		fmt.Print("Can not shutdown with active orders, do you want to shutdown anyway? (y or enter to abort): ")
+		scanner := bufio.NewScanner(os.Stdin)
+		scan := make(chan bool)
+		go func() {
+			scan <- scanner.Scan()
+		}()
+		select {
+		case <-c.ctx.Done():
+			return ok
+		case <-scan:
+		}
+		promptResp := scanner.Text()
+		err := scanner.Err()
+		if err != nil {
+			fmt.Printf("input failed: %v", err)
+			return ok
+		}
+		switch strings.ToLower(promptResp) {
```

Comment:
There doesn't seem to be a need for the `promptResp` variable. 

`switch strings.ToLower(scanner.Text()) {`

Regardless, any call to `scanner.Text()` should go after the error check anyway.